test/bench_cache.lua

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/env tarantool
+
+local ddl = require('ddl')
+local db = require('test.db')
+local clock = require('clock')
+local helper = require('test.helper')
+
+local ITERS = 10000000
+
+local function strcrc32(shard_key)
+    local digest = require('digest')
+    if type(shard_key) ~= 'table' then
+        return digest.crc32(tostring(shard_key))
+    else
+        local crc32 = digest.crc32.new()
+        for _, v in ipairs(shard_key) do
+            crc32:update(tostring(v))
+        end
+        return crc32:result()
+    end
+end
```

Comment:
The same code is existing in `test/helper.lua`. I would reuse it instead of copy-pasting.

Reply:
fixed

Reply:
(Looks resolved, so I'm going to resolve this discussion.)